utils/telldus-core/Makefile

```diff
@@ -0,0 +1,57 @@
+# 
+# Copyright (C) 2010 Telldus Technologies AB
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/nls.mk
+
+PKG_NAME:=telldus-core
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=PeterFromSwe <PeterFromSwe884@gmail.com>
+PKG_LICENSE:=LGPL-2.1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=argp-standalone 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.telldus.com/TellStick/Software/telldus-core/
+PKG_HASH:=a20f6c74814afc23312d2c93ebbb37fdea9deaaee05ae7b6a6275e11e4662014
+
+include $(INCLUDE_DIR)/package.mk
+
+EXTRA_CPPFLAGS:=\
+        -I$(ICONV_PREFIX)/include
+
+EXTRA_CFLAGS:=\
+        -I$(ICONV_PREFIX)/include
+
+define Package/telldus-core
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Telldus TellStick USB interface
+	URL:=http://www.telldus.com
+	DEPENDS:=+confuse +libftdi +libstdcpp +libiconv-full $(ICONV_DEPENDS)
```

Comment:
Does this absolutely require libstdcpp or can uclibc++ be used?

Reply:
Not out of the box. I think iconv and wide char is reason.

Reply:
I just tried. Requires shared_ptr (not in uclibc++).